handler_test.go

```diff
@@ -0,0 +1,90 @@
+package configkit_test
+
+import (
+	. "github.com/dogmatiq/configkit" // can't dot-import due to conflicts
```

Comment:
> // can't dot-import due to conflicts

Does it anyway... ;)

```suggestion
	. "github.com/dogmatiq/configkit"
```

Reply:
Heh, so I've just noticed, something wacky happens with `goimports` sometimes - it seems to move this comment from one import to another :S

Reply:
yeah it does that, does not seem to happen to me if the comment is above the import

Reply:
Yeah, that kinda sucks, because you have put comments on the same line to suppress `vet` warnings about underscore imports in the few cases that they are necessary outside of tests or `package main`.

Reply:
AFAIK, vet does not mind a comment above an underscore import. At least it worked in my case.